src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/ChannelsController.cs

```diff
@@ -52,6 +52,21 @@ public IActionResult ListChannels(string classification = null)
             return Ok(results);
         }
 
+        [HttpGet("{id}/repositories")]
+        [SwaggerApiResponse(HttpStatusCode.OK, Type = typeof(List<string>), Description = "List of repositories in Channel")]
```

Comment:
Not sure if this is the right place to show it, but I'd probably enhance this description mentioning it will only return repositories that have produced a build in the channel, as this won't return any repos that simply consume dependencies from the channel.